components/SearchResults.js

```diff
@@ -2,8 +2,14 @@
 
 const React = require('react');
 
-const SearchResults = ({ results }) => (
-  <ul />
-);
+class SearchResults extends React.Component {
```

Comment:
const is more accurate here as this should be a pure/stateless component. You only create Classes if they hold state.